web/src/LogPane.test.tsx

```diff
@@ -0,0 +1,20 @@
+import React from 'react'
+import ReactDOM from 'react-dom'
+import LogPane from './LogPane'
+import renderer from 'react-test-renderer';
+
+it('renders without crashing', () => {
+  let div = document.createElement('div')
+  Element.prototype.scrollIntoView = jest.fn()
```

Comment:
what's this for?

Reply:
`scrollIntoView` doesn't exist on a DOM elements in the testing environment for some reason? Without this the tests fail with a type error from node complaining that that function doesn't exist.